includes/class-gp-translation-helpers.php

```diff
@@ -335,9 +335,19 @@ public function register_reject_feedback_js( $template, $translation_set ) {
 			'gp-reject-feedback-js',
 			'$gp_reject_feedback_settings',
 			array(
-				'url'         => admin_url( 'admin-ajax.php' ),
-				'nonce'       => wp_create_nonce( 'gp_reject_feedback' ),
-				'locale_slug' => $translation_set['locale_slug'],
+				'url'            => admin_url( 'admin-ajax.php' ),
+				'nonce'          => wp_create_nonce( 'gp_reject_feedback' ),
+				'locale_slug'    => $translation_set['locale_slug'],
+				'reject_reasons' => json_encode(
+					array(
```

Comment:
Can we reference a standard array from a class? Something like `Helper_Translation_Discussion::$reject_reasons`?